docs/docs/09.1-language-tooling.md

```diff
@@ -0,0 +1,81 @@
+---
+id: language-tooling
+title: Language Tooling
+permalink: language-tooling.html
+prev: tooling-integration.html
+next: package-management.html
+---
+
+## ES6 with JSX
+
+### In-browser JSX Transform
+
+If you like using JSX, Babel 5 provided an in-browser ES6 and JSX transformer for development called browser.js that can be included from [CDNJS](http://cdnjs.com/libraries/babel-core/5.8.34). Include a `<script type="text/babel">` tag to engage the JSX transformer.
+
+> Note:
+>
+> The in-browser JSX transformer is fairly large and results in extraneous computation client-side that can be avoided. Do not use it in production — see the next section.
+
+There is also an unofficial/unsupported [Babel6 in-browser transformer](http://www.jimsproch.com/react/), which includes the JSX presets and also enables some of the newer features like `__source` for better debug messages.
+
+### Productionizing: Precompiled JSX
+
+If you have [npm](https://www.npmjs.com/), you can run `npm install -g babel-cli`. Babel has built-in support for React v0.12+. Tags are automatically transformed to their equivalent `React.createElement(...)`, `displayName` is automatically inferred and added to all `React.createClass` calls.
+
+This tool will translate files that use JSX syntax to plain JavaScript files that can run directly in the browser. It will also watch directories for you and automatically transform files when they are changed; for example: `babel --watch src/ --out-dir lib/`.
+
+Beginning with Babel 6, there are no transforms included by default. This means that options must be specified when running the `babel` command, or a `.babelrc` must specify options. Additional packages must also be installed which bundle together a number of transforms, called presets. The most common use when working with React will be to include the `es2015` and `react` presets. More information about the changes to Babel can be found in [their blog post announcing Babel 6](http://babeljs.io/blog/2015/10/29/6.0.0/).
+
+Here is an example of what you will do if using ES2015 syntax and React:
+
+```
+npm install babel-preset-es2015 babel-preset-react
+babel --presets es2015,react --watch src/ --out-dir lib/
+```
+
+By default JSX files with a `.js` extension are transformed. Run `babel --help` for more information on how to use Babel.
+
+Example output:
+
+```
+$ cat test.jsx
```

Comment:
Yeah, that example was literally just moved verbatim from the existing `tooling-integration` page.  I don't feel strongly though, so I'll change it.